src/client/lixa_tx.c

```diff
@@ -53,6 +53,23 @@
 #endif /* LIXA_TRACE_MODULE */
 #define LIXA_TRACE_MODULE   LIXA_TRACE_MOD_CLIENT_TX
 
+static const char *tx_open_profile = NULL;
```

Comment:
LIXA implementation of TX Demarcation Specification supports multithreading and this global variable would be a flaw: the variable must be put in a thread dedicated status structure (see other notes to get the idea).